apps/profiles/models.py

```diff
@@ -31,6 +35,8 @@ class Instructor(models.Model):
 
     group = models.ForeignKey('groups.Group', related_name='instructor_members', null=True, blank=True, on_delete=models.PROTECT)
 
+    date_promoted = models.DateTimeField(auto_now_add=True)
```

Comment:
What does this field represent?

Reply:
This is the day that a user became an instructor. Is `created` more straight-forward?

Reply:
It's used for metrics. Resolving this based on our verbal agreement.